packages/builder/cypress.env.json

```diff
@@ -0,0 +1,10 @@
+{
+"HOST_IP": "",
+"oracle": {
```

Comment:
We'll want to move these creds out of the repo, preferably injected as env variables